config/product-mapping.js

```diff
@@ -1,4 +1,4 @@
-const sevenDayDelivery = [
+const sevenDay = [
```

Comment:
Kanske vore en idé att lägga de här i ett objekt?
```suggestion
const deliveryConfig = {
  sevenDay: [...],
  sixDay: [...],
  ...
}
```
Men bara ett förslag.

Reply:
Tänkte tanken, valde variabler 😇